wrapper/src/main/java/software/amazon/jdbc/hostavailability/PassiveExponentialBackoffHostAvailabilityStrategy.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.hostavailability;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Properties;
+
+public class PassiveExponentialBackoffHostAvailabilityStrategy implements HostAvailabilityStrategy {
+  private int maxRetries = 5;
+  private int baseBackoffTimeSeconds = 30;
+  private int notAvailableCount = 0;
+  private Timestamp lastChanged;
+
+  public PassiveExponentialBackoffHostAvailabilityStrategy(Properties props) {
+    if (HOST_AVAILABILITY_STRATEGY_MAX_RETRIES.getInteger(props) >= 1) {
+      this.maxRetries = HOST_AVAILABILITY_STRATEGY_MAX_RETRIES.getInteger(props);
+    }
+    if (HOST_AVAILABILITY_STRATEGY_BASE_BACKOFF_TIME.getInteger(props) >= 1) {
+      this.baseBackoffTimeSeconds = HOST_AVAILABILITY_STRATEGY_BASE_BACKOFF_TIME.getInteger(props);
+    }
+    lastChanged = Timestamp.from(Instant.now());
+  }
+
+  @Override
+  public void setHostAvailability(HostAvailability hostAvailability) {
+    this.lastChanged = Timestamp.from(Instant.now());
+    if (hostAvailability == HostAvailability.AVAILABLE) {
+      this.notAvailableCount = 0;
+    } else if (hostAvailability == HostAvailability.NOT_AVAILABLE) {
```

Comment:
```suggestion
    } else {
```
Since there are only two states now

Reply:
Done!